Anchor.toml

```diff
@@ -11,7 +11,7 @@ url = "https://anchor.projectserum.com"
 
 [provider]
 cluster = "localnet"
-wallet = "/Users/dhruvjain/.config/solana/id.json"
+wallet = "~/.config/solana/id.json"
```

Comment:
this is important, so that it also works on my machine

Reply:
Yes this is important